command/sub8_alarm/sub8_alarm/alarm_handlers/thruster_out.py

```diff
@@ -11,16 +11,23 @@ def __init__(self):
         self.update_layout = rospy.ServiceProxy('update_thruster_layout', UpdateThrusterLayout)
 
     def drop_thruster(self, thruster_name):
-        self.dropped_thrusters.append(thruster_name)
+        if thruster_name not in self.dropped_thrusters:
+            self.dropped_thrusters.append(thruster_name)
         # Drop a single thruster
         rospy.logwarn("Dropping thuster {}".format(thruster_name))
         # a priori: Each thruster name is a string, do not need to map(str, remaining_thursters)
         rospy.logwarn("Dropped thrusters: " + ', '.join(self.dropped_thrusters))
 
-    def handle(self, time_sent, parameters):
+    def handle(self, time_sent, parameters, alarm_name):
         self.drop_thruster(parameters['thruster_name'])
         self.update_layout(self.dropped_thrusters)
 
```

Comment:
Yep, fixed now.
The param is being passed to the function by the handler so python will complain if there's a missing argument that's not actually there